src/SonarQube.Client/Api/Requests/V5_20/GetRoslynExportProfileRequest.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.IO;
+using Newtonsoft.Json;
+using SonarQube.Client.Messages;
+
+namespace SonarQube.Client.Api.Requests.V5_20
+{
+    public class GetRoslynExportProfileRequest : RequestBase<RoslynExportProfileResponse>, IGetRoslynExportProfileRequest
+    {
+        [JsonProperty("language")]
+        public string LanguageKey { get; set; }
+
+        [JsonProperty("name")]
+        public virtual string QualityProfileName { get; set; }
+
+        [JsonProperty("organization")]
+        public virtual string OrganizationKey { get; set; }
+
+        [JsonProperty("exporterKey", Required = Required.Always)]
+        public string ExporterKey => $"roslyn-{LanguageKey}";
+
+        protected override string Path => "api/qualityprofiles/export";
+
+        protected override RoslynExportProfileResponse ParseResponse(string response)
+        {
+            using (var reader = new StringReader(response))
+            {
+                // TODO: consider not returning the xml directly
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)